libs/eo/shared/atomic-design/ui-atoms/src/lib/eo-date-picker/eo-date-picker-dialog.component.ts

```diff
@@ -14,6 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/* eslint-disable @nrwl/nx/enforce-module-boundaries */
```

Comment:
Why did this work before? Question if it is needed now.

Reply:
Like i said, this is because of how NX thinks the dependencies work :) Just getting around it. :)